docs/angular-ngrx.md

```diff
@@ -64,31 +64,29 @@ Config for **Angular** projects using **NgRx** library.
 | [![rxjs](./icons/other/rxjs.png)](https://github.com/cartant/eslint-plugin-rxjs#readme) | [no-unsafe-first](https://github.com/cartant/eslint-plugin-rxjs/tree/main/docs/rules/no-unsafe-first.md)<br />Forbids unsafe `first`/`take` usage in effects and epics.              |         |         |           |
 | [![rxjs](./icons/other/rxjs.png)](https://github.com/cartant/eslint-plugin-rxjs#readme) | [no-unsafe-switchmap](https://github.com/cartant/eslint-plugin-rxjs/tree/main/docs/rules/no-unsafe-switchmap.md)<br />Forbids unsafe `switchMap` usage in effects and epics.         |         |         |           |
 
-### ⚠️ Warnings (24)
```

Comment:
A couple of rules were removed. I guess that wasn't intentional?

Judging by your latest commit, it seems like you misunderstood https://github.com/code-pushup/eslint-config/pull/23#discussion_r1787676503 and removed the `warn`s instead of the `off`s.

![image](https://github.com/user-attachments/assets/008f787a-a5ba-493c-a154-4d29eb0f7a85)

Reply:
Yes I acted like a robot and just removed the green highlighted lines